Replace usage of `\htmlonly` with `<div>`

albert-github opened the pull request:
The HTML `<div>` construct with attributes is supported by doxygen (in HTML, on other places it is ignored) and can theerfore be removed.
- the advantage is that a mismatch of `<div>` / `</div>` is notice (e.g. when they are in different `ALIASES` and one of the 2 has been forgotten, see also #8560)
- the `ALIASES`: `cgalModifBegin` and `cgalModifEnd` are not used (anymore) and are therefore removed

albert-github committed:
Remove usage of `\htmlonly` with `<div>`

The HTML `<div>` construct with attributes is supported by doxygen (in HTML, on other places it is ignored) and can theerfore be removed.
- the advantage is that a mismatch of `<div>` / `</div>` is notice (e.g. when they are in different `ALIASES` and one of the 2 has been forgotten, see also #8560)
- the `ALIASES`: `cgalModifBegin` and `cgalModifEnd` are not used (anymore) and are therefore removed

afabri commented:
What do you mean with that `cgalModifBegin` are not used. They do not appeal in the master branch but they are useful when proposing changes in the manual. So they are just needed for "development".

albert-github committed:
Remove usage of `\htmlonly` with `<div>`

After review:
- reinstated the `ALIASES`: `cgalModifBegin` and `cgalModifEnd` that might be used during development.

albert-github commented:
Reinstated the"modif" `ALIASES`.

- what to do with the `\latexonly` part ?
- interesting question is are these 2 `ALIASES` filtered out when a release is made so they won't accidentally slip into a release?

albert-github committed:
Remove usage of `\htmlonly` with `<div>`

Another place where `\htmlonly` can be removed

afabri commented:
I think we could give up `\latexonly`  as we have no ambition to create something like pdf.  @MaelRL any opinion?

sloriot commented:
Successfully tested in CGAL-6.1-Ic-50

albert-github commented:
@sloriot @lrineau @afabri 

Any chance that this will be integrated in CGAL or is there something further to be done here

albert-github committed:
Merge branch 'main' into feature/bug_htmlonly

# Conflicts:
#	Documentation/doc/resources/1.14.0/BaseDoxyfile.in

Remove `\latexonly` as well

albert-github commented:
Merged and removed `\latexonly` as well
